wrappers/python/tests/anoncreds/test_prover_create_proof.py

```diff
@@ -42,29 +44,33 @@
     }
 
     schemas = {
-        claim_for_attr: anoncreds.get_gvt_schema_json(1)
+        claim_for_attr: gvt_schema
     }
 
     claim_defs = {
-        claim_for_attr: json.loads(init_common_wallet[1])
+        claim_for_attr: json.loads(claim_def_json)
     }
 
-    await prover_create_proof(init_common_wallet[0], json.dumps(proof_req), json.dumps(requested_claims),
-                              json.dumps(schemas), anoncreds.COMMON_MASTER_SECRET_NAME,
+    await prover_create_proof(wallet_handle, json.dumps(proof_req), json.dumps(requested_claims),
```

Comment:
Should we check that returned proof satisfies the expected result (as defined by the API)?